recipe/meta.yaml

```diff
@@ -8,8 +8,7 @@ source:
     md5: 3e1c451c087f8462955426da38aa3b3d
 
 build:
-    number: 2
-    skip: True  # [win and py35]
```

Comment:
It seems that https://github.com/conda-forge/libtiff-feedstock/pull/6 fixed this.